tomcat/tests/test_check.py

```diff
@@ -0,0 +1,18 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from .common import TOMCAT_E2E_METRICS
+
+
+@pytest.mark.e2e
+def test(dd_agent_check):
+    instance = {}
+    aggregator = dd_agent_check(instance)
```

Comment:
Have you tried adding:
```suggestion
    aggregator = dd_agent_check(instance, rate=True)
```
it might help get some more metrics

Reply:
Adding this does not add any more metrics unfortunately. Florian and I tried this in the Kafka e2e PR but this is likely related to this unreleased JMX PR:

https://github.com/DataDog/jmxfetch/pull/267

Reply:
Ah, makes sense!